opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -251,6 +253,31 @@ def get_label_set(self, labels: Dict[str, str]) -> "LabelSet":
         return DefaultLabelSet()
 
 
+class DefaultMeter(Meter):
+    """The default Meter used when no Meter implementation is available."""
+
+    def record_batch(
+        self,
+        label_set: LabelSet,
+        record_tuples: Sequence[Tuple["Metric", ValueT]],
+    ) -> None:
+        pass
+
+    def create_metric(
+        self,
+        name: str,
+        description: str,
+        unit: str,
+        value_type: Type[ValueT],
+        metric_type: Type[MetricT],
+        label_keys: Sequence[str] = (),
+        enabled: bool = True,
+        monotonic: bool = False,
+    ) -> "Metric":
+        # pylint: disable=no-self-use
+        return DefaultMetric()
+
+
```

Comment:
I think `get_label_set` will be needed too.

Reply:
Fixed, i'd forgotten to make `get_label_set` abstract.